jupyter_server/serverapp.py

```diff
@@ -1415,6 +1415,8 @@ def init_server_extensions(self):
                     func = getattr(mod, 'load_jupyter_server_extension', None)
                     if func is not None:
                         func(self)
+                    # Add debug log for loaded extensions.
+                    self.log.debug("%s is enabled and loaded.", modulename))
```

Comment:
These should be indented. 
 
We should probably add an `else:` clause that logs a **warning** stating that the extension, while enabled, doesn't have a `load_jupyter_server_extension()` method.

Reply:
Good catch!

:+1: Warning added.